src/styles/variables.css

```diff
@@ -22,6 +22,8 @@
   --grey-blue-1: rgb(232, 236, 245);
   --grey-blue-2: rgb(244, 246, 250);
   --grey-blue-3: rgb(246, 246, 246);
+  --action-secondary: rgb(76, 93, 255);
+
```

Comment:
Please remove the extra new line here

Reply:
This is why `stylelint` is failing your builds

![Screenshot from 2020-12-08 15-59-09](https://user-images.githubusercontent.com/1193222/101493179-5f1f3a80-396e-11eb-92e8-d6db3cf67be2.png)